content/en/blog/2018/egress-tcp/index.md

```diff
@@ -339,7 +339,7 @@ bypassing the sidecar proxies.
 ## Relation to mesh expansion
```

Comment:
Should this title say "mesh expansion" or "VM Support" ?

Reply:
We don't typically update blog posts as things change. That's why I left the text here unchanged. I can definitely change it if you think it would lead to confusion @ssuchter

Reply:
Oh, I didn't notice this was editing blog text. Shouldn't need to do that. If the only reason to edit the blog at all is to fix the link, then let's not change the text.